CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Updated
```

Comment:
```suggestion
### Changed
```

We want to match the format described here: https://keepachangelog.com/en

Reply:
I do this every time I swear...